sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch.md

```diff
@@ -0,0 +1,288 @@
+        "Top hotels in town",
+        new SearchOptions
+        {
+            Vector = new() { Value = vectorizedResult, KNearestNeighborsCount = 3, Fields = "DescriptionVector" },
+        });
+
+int count = 0;
+Console.WriteLine($"Simple Hybrid Search Results:");
+await foreach (SearchResult<Hotel> result in response.GetResultsAsync())
+{
+    count++;
+    Hotel doc = result.Document;
+    Console.WriteLine($"{doc.HotelId}: {doc.HotelName}");
+}
+Console.WriteLine($"Total number of search results:{count}");
+```
+
+#### Semantic Hybrid Search
+
+To use semantic search, we need to add a `SemanticConfiguration` to the index. In this example, we will update the previously created index and add the `SemanticConfiguration` to it.
+
+```C# Snippet:Azure_Search_Documents_Tests_Samples_Sample07_Vector_Search_Semantic_Index
+string indexName = "Hotel";
+SearchIndex createdIndex = await indexClient.GetIndexAsync(indexName);
+
+createdIndex.SemanticSettings = new()
+{
+    Configurations =
+    {
+           new SemanticConfiguration("my-semantic-config", new()
+           {
+               TitleField = new(){ FieldName = "HotelName" },
+               ContentFields =
+               {
+                   new() { FieldName = "Description" }
+               },
+               KeywordFields =
+               {
+                   new() { FieldName = "Category" }
+               }
+           })
+    }
+};
+
+// Update index
+await indexClient.CreateOrUpdateIndexAsync(createdIndex);
+```
+With the semantic configuration added, we can now execute a semantic hybrid query.
+
+```C# Snippet:Azure_Search_Documents_Tests_Samples_Sample07_Semantic_Hybrid_Search
+IReadOnlyList<float> vectorizedResult = VectorSearchEmbeddings.SearchVectorizeDescription; // "Top hotels in town"
+
+SearchResults<Hotel> response = await searchClient.SearchAsync<Hotel>(
+    "Is there any hotel located on the main commercial artery of the city in the heart of New York?",
+    new SearchOptions
+    {
+        Vector = new SearchQueryVector { Value = vectorizedResult, KNearestNeighborsCount = 3, Fields = "descriptionVector" },
+        QueryType = SearchQueryType.Semantic,
+        QueryLanguage = QueryLanguage.EnUs,
+        SemanticConfigurationName = "my-semantic-config",
+        QueryCaption = QueryCaptionType.Extractive,
+        QueryAnswer = QueryAnswerType.Extractive,
+    });
+
+int count = 0;
+Console.WriteLine($"\nSemantic Hybrid Search Results:");
+
+Console.WriteLine($"\nQuery Answer:");
+foreach (AnswerResult result in response.Answers)
+{
+    Console.WriteLine($"Answer Highlights: {result.Highlights}");
+    Console.WriteLine($"Answer Text: {result.Text}");
+}
+
+await foreach (SearchResult<Hotel> result in response.GetResultsAsync())
+{
+    count++;
+    Hotel doc = result.Document;
+    Console.WriteLine($"\nHotelId: {doc.HotelId} \n HotelName: {doc.HotelName} \n Category: {doc.Category} \n Description: {doc.Description}");
```

Comment:
Long lines (also line 250 above) don't render well and make the customer scroll horizontally, which can be a pain with many mice. I'd use separate `Console.WriteLine` calls e.g.,
```
Console.WriteLine();
Console.WriteLine($"HotelId: {doc.HotelId}");
Console.WriteLine($" HotelName: {doc.HotelName}");
...
```
Also, was the single space before each property except the initial `HotelId` intentional?

Reply:
Updated the Console.WriteLine to maintain the consistency with the other sample codes.